modules/react/popup/lib/PopupCard.tsx

```diff
@@ -71,12 +70,11 @@ export const PopupCard = createSubcomponent('div')({
       as={As}
       transformOrigin={transformOrigin}
       position="relative"
-      padding="l"
       depth={5}
       maxWidth={`calc(100vw - ${space.l})`}
-      spacing={0}
```

Comment:
We can keep the underlying element a `Stack` however, if we keep this spacing="0" this will remove marginTop on the first element in the stack. I suggest we either remove this prop or change the underlying element to be `Flex`

Reply:
We're changing PopupCard to use `Flex` instead of `Stack`. `Stack` doesn't work conceptually given the presence of the Popup.CloseIcon within the container (CloseIcon doesn't make sense as a `Stack` child).